manifests/0000_20_cluster-openshift-apiserver-operator_07_deployment.yaml

```diff
@@ -24,19 +24,20 @@ spec:
         command: ["cluster-openshift-apiserver-operator", "operator"]
         args:
         - "--config=/var/run/configmaps/config/config.yaml"
-        - "-v=4"
+        - "-v=2"
         volumeMounts:
         - mountPath: /var/run/configmaps/config
           name: config
-        - mountPath: /var/run/secrets/serving-cert
+        - mountPath: /var/run/secrets/serving-certs
```

Comment:
pretty sure I didn't pluralize this in the defaults.  Did I?

Reply:
> pretty sure I didn't pluralize this in the defaults. Did I?

yeah, not plural https://github.com/openshift/library-go/blob/master/pkg/config/configdefaults/config_default.go#L48